Makefile

```diff
@@ -703,10 +703,11 @@ prepare-merge-fetch:
 	components_var="REMOTE_COMPONENTS_$${GIT_REMOTE//-/_}"; \
 	[ -n "$${!components_var}" ] && REPOS="`echo $${!components_var} | sed 's@^\| @ $(SRC_DIR)/@g'`"; \
 	for REPO in $$REPOS; do \
-		$$SCRIPT_DIR/get-sources || exit 1; \
+		"$$SCRIPT_DIR/get-sources" || exit 1; \
 	done
 
 prepare-merge: prepare-merge-fetch show-unmerged
+merge: prepare-merge-fetch do-merge
```

Comment:
This is almost the same as `make get-sources`, do we need it?

Reply:
The difference is that `make get-sources` switches everything to `master`, whereas this (intends to) merge `master` into one’s local changes.

Reply:
Ok, for the cases when actual merge is intended. This actually makes sense to produce merge on `make merge`, but avoid it on `make get-sources`.